application/templates/viewdeed.html

```diff
@@ -5,25 +5,95 @@
-        <li>MD Ref: {{deed_data.deed.md_ref}}</li>
-    </ul>
-
-    <form action="/searchdeed">
+
+    <h1 class="heading-xlarge">Your mortgage deed</h1>
+
+    <div class="text">
+      <div class="alert-inline">
+        <p><b>This is a legal document &mdash; please check all details carefully.</b></p>
+      </div>
+    </div>
+    <div class="column-two-thirds">
+    <article class="deed-document--bounds">
+        <h2 class="heading-large">Electronic Mortgage Deed</h2>
+            <h2 class="heading-medium">Property: </h2>
+                <p itemscope itemtype="http://schema.org/PostalAddress">
+                <span itemprop="streetAddress">
+                  Address Line 1<br>
+                </span>
+                <span itemprop="addressLocality">Locality</span><br>
+                <span itemprop="postalCode">PO57 CDE</span>
+              </p>
+
+            <dl class="definition-inline">
+            <dt><b>Title number</b></dt>
+                <dd>{{ deed_data.deed.title_number }}</dd>
+            </dl>
+
+          <h2 class="heading-medium">Lender</h2>
+        <div class="text">
+            Lender Information will go here
+        </div>
+
+        <h2 class="heading-medium">Borrower</h2>
+        <div class="grid-row">
+            {% for borrower in deed_data.deed.borrowers %}
+            <div class="column-third">
+              <p>
+                <b><a>{{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
+                borrower.surname}}</a></b><br>
+              </p>
+            </div>
+            {% endfor %}
+        </div>
+
+        <div class="text">
+
+            <h3 class="heading-small">Charging clause:</h3>
+            <p>The borrower, with full title guarantee, charges to the lender the property by way of legal mortgage with payment of all money secured by this charge.</p>
+
+            <h3 class="heading-small">Additional provisions:</h3>
+            <ol>
+              <li>
+                <p>This Mortgage Deed incorporates the <a href="#" rel="external">Lenders Mortgage Conditions and Explanation 2006</a>, a copy of which the borrower has received.</p>
+              </li>
+              <li>
+                <p>The lender is obliged to make further advances and applies for the obligation to be entered in the register.</p>
+              </li>
+              <li>
+                <p>The borrower applies to enter a restriction in the register that no disposition of the registered estate by the proprietor of the registered estate is to be registered without a written consent signed by the proprietor for the time being of the charge dated [the date of this charge] in favour of Bank of England Plc referred to in the charges register.</p>
+              </li>
+            </ol>
+        </div>
+        <p class="text">
+
+              <label class="heading-small" for="auth-code">Effective date and time</label></br>
```

Comment:
Are these labels going to have something associated with them later? The for attributes don't relate to any input fields...